scripts/devnet.yaml

```diff
@@ -8,14 +9,14 @@ cronos_777-1:
   validators:
     - coins: 1000000000000000000stake,1000000000000000000basetcro
       staked: 1000000000000000000stake
-      mnemonic: visit craft resemble online window solution west chuckle music diesel vital settle comic tribe project blame bulb armed flower region sausage mercy arrive release
+      mnemonic: ${VALIDATOR1_MNEMONIC}
```

Comment:
Is there a simple way to use this config directly for development after this change?

Reply:
Maybe we can support the expansion feature in `pystarport` first?

Reply:
Yes, I already support it in local pystarport. It is ok to summit another PR for it.